packages/core/src/app/checkout/CheckoutApp.tsx

```diff
@@ -55,23 +40,20 @@ export default class CheckoutApp extends Component<CheckoutAppProps> {
 
     render() {
         return (
-            <ErrorBoundary logger={this.errorLogger}>
```

Comment:
Howcome we are removing the `ErrorBoundary` here?

Reply:
Ignore I see it's moved to the child component.

Reply:
Moved to `checkout.tsx`, that's when we can have a read on the feature flag.
And I don't think we'll compromise on the capture scope, maybe a little.
I was going to comment here, another method is to keep the `ErrorBoundary` here and set it to always 100%. Only use the feature flag to control the error logger in `checkout.tsx`.